config/feature_flags.yml

```
@@ -34,4 +34,9 @@ feature_flags:
       Allow starting an application on this service directly after completing
       an eligibility check.
 
+  show_all_applicants:
```

```suggestion
  show_all_applications_filter:
```

Just a suggestion to make the name a little more explicit.